.github/pull_request_template.md

```diff
@@ -10,3 +10,4 @@ Description of the pull request changes and motivation.
 - [ ] Integration tests added.
 - [ ] This change requires new documentation.
   - [ ] Documentation has been added/updated.
+  - [ ] Changelog has been updated.
```

Comment:
This should suffice for now. A later refinement would be a pipelines that rejects the merge if it wasn't.